Directory.Build.props

```diff
@@ -36,7 +36,7 @@
 
     <!-- Version information -->
     <ScalarVersion>0.2.173.2</ScalarVersion>
-    <GitPackageVersion>2.20200227.1</GitPackageVersion>
+    <GitPackageVersion>2.20200317.1-pr</GitPackageVersion>
     <MinimumGitVersion>v2.25.0.vfs.1.1</MinimumGitVersion>
```

Comment:
You are right to check. I believe we are still good, and here's why:

This minimum is really only for the GVFS protocol integration. There are some things where we hope the partial clone integration is _better_ in later version, this is only to ensure the command-line interface and config settings don't have breaking changes to what we expect. Leaving this as old as possible maximizes the ability for VFS for Git to be installed side-by-side.

The only change we made that could be considered a dependence on new behavior is the `fsmonitor` hook version 2. However, the `core.fsmonitorHooksVersion` config option did not exist before v2.26.0, so even though we set that option an old Git would not see it. Since we copy the hook from the installed directory, we know that hook is compatible with the current version.

Thanks! I'll consider adding a comment here to clarify our purpose with this minimum version.